widget/ui/src/components/Toast/Toast.helpers.ts

```diff
@@ -5,3 +5,6 @@ export function idGenerator() {
     return id;
   };
 }
+
+export const CLOSED_DELAY = 400;
+export const OPEN_DELAY = 10;
```

Comment:
Can you explain why we need "OPEN_DELAY"? I removed it and it worked properly. Also the other variable is named "CLOSED_DELAY " but it is used as transition duration so it is not just for closing the toast and it is used for opening it too. So I think you can improve the name of that variable.